src/setupProxy.js

```diff
@@ -0,0 +1,12 @@
+const fs = require("fs");
+
+module.exports = function (app) {
+  app.use("/configOverride", (_req, res, _next) => {
```

Comment:
I don't like to use camel case in url, maybe:
```suggestion
  app.use("/config.override.json", (_req, res, _next) => {
```